.pre-commit-config.yaml

```diff
@@ -32,3 +32,9 @@ repos:
       - id: ruff
         args: ['--fix']
       - id: ruff-format
+
+  - repo: https://github.com/rstcheck/rstcheck
+    rev: v6.2.4
+    hooks:
+      - id: rstcheck
+        args: ['--config', '.rstcheck.cfg']
```

Comment:
I wonder if we should use the command line arguments instead of the config file (there are not too much of it), to avoid adding a new file.

Reply:
No strong feelings on this!